webpack/ForemanColumnExtensions/index.js

```diff
@@ -97,6 +99,47 @@ const hostsIndexColumnExtensions = [
     weight: 2400,
     isSorted: true,
   },
+  {
+    columnName: 'content_view_environments',
+    title: __('Content view environments'),
+    wrapper: (hostDetails) => {
+      const contentViewEnvironments =
+        hostDetails?.content_facet_attributes?.content_view_environments ?? [];
+      if (contentViewEnvironments.length === 0) return '—'; // don't show popover
+      return (
+        <Flex>
+          {contentViewEnvironments.length > 1 &&
+            <FlexItem>
+              <Badge isRead>{contentViewEnvironments.length}</Badge>
+            </FlexItem>
+          }
+          <Popover
+            id="content-view-environments-tooltip"
+            className="content-view-environments-tooltip"
+            maxWidth="34rem"
+            headerContent={hostDetails.display_name}
+            bodyContent={
+              <Flex direction={{ default: 'column' }}>
+                {contentViewEnvironments.map(env => (
+                  <ContentViewEnvironmentDisplay
+                    key={`${env.lifecycle_environment.name}-${env.content_view.name}`}
+                    contentView={env.content_view}
+                    lifecycleEnvironment={env.lifecycle_environment}
+                  />
+                ))}
+              </Flex>
+            }
+          >
+            <FlexItem>
+              {truncate(contentViewEnvironments.map(cve => cve.candlepin_name).join(', '), 35)}
```

Comment:
❤️ Filed a bug for CV UI in other places cause we do not truncate these names everywhere..
https://projects.theforeman.org/issues/37530